athf/mcp/server.py

```
@@ -52,9 +52,11 @@ def _discover_plugin_tools() -> list:
     """Discover MCP tool registration functions from installed plugins."""
     if sys.version_info >= (3, 10):
         from importlib.metadata import entry_points
+
         return list(entry_points(group="athf.mcp_tools"))
     else:
```

Ruff (0.16.1)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)